Subsonicious/Subsonicious/SceneDelegate.swift

```diff
@@ -45,6 +45,7 @@ final class SceneDelegate: UIResponder, UIWindowSceneDelegate {
             .environmentObject(authenticationManager)
             .environmentObject(player)
             .environmentObject(playerObserver)
+            .environmentObject(nowPlayingInfoManager)
```

Comment:
maybe we could have a little system to abstract all these environment object ? maybe an extension to do 
`myView.managers(allManagersArray)` but it would make an array of all manager so not really good after all...

Reply:
Yep, I think separated instances are easier to manage than an array :)